cmake/modules/re2.cmake

```diff
@@ -87,9 +89,8 @@ else()
 
 	install(DIRECTORY ${RE2_SRC}/lib/ DESTINATION "${CMAKE_INSTALL_LIBDIR}/${LIBS_PACKAGE_NAME}"
 			COMPONENT "libs-deps"
-			FILES_MATCHING PATTERN "libre2*")
-	install(DIRECTORY "${RE2_INCLUDE}" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${LIBS_PACKAGE_NAME}"
-			COMPONENT "libs-deps")
+			FILES_MATCHING PATTERN ${RE2_LIB_PATTERN})
+	# Includes are internal-only.
```

Comment:
While this is true, there are more deps that are internal but we install headers for.
I'd leave that here since it causes no damage.